notebooks/233-blip-visual-language-processing/233-blip-quantize-nncf.ipynb

```diff
@@ -0,0 +1,724 @@
+{
```

Comment:
<p>if I right understand, you notebook has 2 different actions - quantize model which working with IR and compress weights which work with original model. Can we reduce required memory for running initialize only model when they need. Intialize original model only when you compress weights, release compiled model which used for preparing quantization dataset (as you no need in it anymore if it Is possible)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Switched to OpenVINO backend for weight compression